libs/styles/rounded-corner.css

```diff
@@ -0,0 +1,29 @@
+:root {
```

Comment:
nit: Since the classnames are `x-rounded-corners` the filename should probably be `rounded-corners.css`